crc/services/approval_service.py

```diff
@@ -84,6 +92,45 @@ def update_approval(approval_id, approver_uid, status):
             db_approval.status = status
             session.add(db_approval)
             session.commit()
+            if status == ApprovalStatus.APPROVED.value:
+                # second_approval = ApprovalModel().query.filter_by(
+                #     study_id=db_approval.study_id, workflow_id=db_approval.workflow_id,
+                #     status=ApprovalStatus.PENDING.value, version=db_approval.version).first()
+                # if second_approval:
+                    # send rrp approval request for second approver
+                ldap_service = LdapService()
+                pi_user_info = ldap_service.user_info(model.study.primary_investigator_id)
+                approver_info = ldap_service.user_info(approver_uid)
+                # send rrp submission
+                send_ramp_up_approved_email(
+                    'askresearch@virginia.edu',
+                    [pi_user_info.email_address],
+                    f'{approver_info.display_name} - ({approver_info.uid})'
+                )
+            elif status == ApprovalStatus.DECLINED.value:
+                ldap_service = LdapService()
+                pi_user_info = ldap_service.user_info(model.study.primary_investigator_id)
+                approver_info = ldap_service.user_info(approver_uid)
+                # send rrp submission
+                send_ramp_up_denied_email(
+                    'askresearch@virginia.edu',
+                    [pi_user_info.email_address],
+                    f'{approver_info.display_name} - ({approver_info.uid})'
+                )
+                first_approval = ApprovalModel().query.filter_by(
+                    study_id=db_approval.study_id, workflow_id=db_approval.workflow_id,
+                    status=ApprovalStatus.APPROVED.value, version=db_approval.version).first()
```

Comment:
Should this have an "order_by=id" on it - I don't think there is a guarantee that it will be returned in any order unless we specify it.

Reply:
But I see it does look for status of Approved, so this should just work anyway - at least in the two approvals case.

Reply:
Yes, that's the idea, as long as the status are respected, it should work